README.md

```diff
@@ -1,2 +1,89 @@
-# final-capstone-backend
-Final capstone backend
+# Easy booking
```

Comment:
- [ ] In your `spec` folder, you have som auto generated unused "pending" tests.  Kindly delete the ones that you are not using.